### PR TITLE
feat(vscode): Sort target names alphabetically

### PR DESCRIPTION
**Pull request by pietdevries94**
Implements #1959 
I checked if it works with run target and in the nx panel. Both are sorted alphabetically.

### CODE REVIEW

**Comment by MaxKless** [comment]
Sorry for the slow communication. This looks good, I'll have a look at it soon to see if there's anything we missed.
Appreciate the PR a lot!!

File: libs/vscode/nx-cli-quickpicks/src/lib/select-run-information.ts
Comment (MaxKless) [inline libs/vscode/nx-cli-quickpicks/src/lib/select-run-information.ts:1]: also sort in the other if condition:

```
if (projectName) {
    return Object.keys(workspace.projects[projectName].targets || {}).sort();
  }
  ```

**Review by MaxKless — CHANGES_REQUESTED** [review]
one sort is missing, otherwise it looks good. 

For the project view, I would not sort the projects for now. Since the project view is kind of a visualisation of project.json files, it makes sense to keep the same order.

But for the run target view & quickpicks, it's a good idea. So just one small change.

Do you have access to IntelliJ? Because we'll need to implement the changes there too.

**Comment by pietdevries94** [comment]
I did the two changes as requested :) 

I sadly don't have an IntelliJ setup and I also haven't written any Java for like 15 years, so I don't feel as comfortable to make the same change at that side.

I however have enabled edits on this PR so if you want to do it, feel free to add it here. And if someone else wants to do it, feel free to ask, I'm fine with inviting people on that repo

**Pull request merged**